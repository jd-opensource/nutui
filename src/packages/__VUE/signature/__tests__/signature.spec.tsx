// import { mount } from s

// TODO
test('需配置 Vitest Canvas Mock', () => {
  expect(1).toEqual(1);
});
// test('props custom-class', async () => {
//   const wrapper = mount(() => {
//     return <Signature customClass="test" line-lineWidth={5} strokeStyle="#333"></Signature>;
//   });
//   const signatureWrapper = wrapper.find('.nut-signature');
//   expect(signatureWrapper.classes()).toContain('test');

//   const canvas = createCanvas(400, 400);
//   const ctx = canvas.getContext('2d');
//   ctx.lineWidth = 5;
//   ctx.strokeStyle = '#333';
//   ctx.beginPath();
//   ctx.lineTo(10, 50);
//   ctx.lineTo(80, 120);
//   ctx.stroke();
//   const _img = document.createElement('img');
//   _img.src = canvas.toDataURL();
//   expect(canvas.toDataURL()).toMatchSnapshot();
// });
