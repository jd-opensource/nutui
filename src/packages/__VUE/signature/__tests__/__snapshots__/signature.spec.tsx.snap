// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`props custom-class 1`] = `"data:image/png;base64,00"`;
